glean-core/ios/Glean/Config/Configuration.swift

```diff
@@ -2,6 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/// The Configuration struct describes how to configure Glean
```

Comment:
Does swift also has special handling for "doc comments" (3 slashes)?
Does it come with a nice doc renderer?

Reply:
See [here](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/index.html) for more info on the markdown style handling I was using for reference.

Reply:
Basically it uses standard markdown with a few special thing to denote sections like `- parameters:`, etc...

Instead of the triple slash, we can also use:
```Swift
/** 
... 
*/
```

Reply:
triple-slash it is!